aunt-leahs-app/src/components/SelectBox.jsx

```diff
@@ -22,15 +22,15 @@ const SelectBox = ({ name, items, onSelectItem }) => {
                     <MenuItem value=''>
                         <em>None</em>
                     </MenuItem>
-                    {items.map(item =>
-                        <MenuItem key={item.id} value={item.id}>
+                    {this.state.items.map(item =>
```

Comment:
There's a runtime error happening here. this.state should be remove for the page to load.